pkg/interfaces/contracts/solidity-utils/helpers/IAuthentication.sol

```diff
@@ -2,14 +2,15 @@
 
 pragma solidity ^0.8.24;
 
+/// @notice Simple interface for permissioned calling of external functions.
```

Comment:
Just to make sure, we apply this pattern instead of `/* */` when it's a one-liner explanation of the function, is that it?

Reply:
Yes; "///" means a one-liner.